queries.yaml

```diff
@@ -177,6 +177,23 @@ post_process:
         14: { area:  100, volume: 200000 }
         15: { area:  100, volume: 100000 }
 
+  - fn: vectordatasource.transform.build_fence
+    params:
+      base_layer: landuse
+      start_zoom: 16
+      prop_transform:
+        kind: true
+        id: true
+        source: true
+        area: true
+        min_zoom: true
```

Comment:
Is there a difference between this list of properties and the source feature? If not, would it be better to just make a copy of all the source properties instead of specifying them here? That way if we add a property to landuse we don't have to remember to add it in here.